### PR TITLE
Colin Narans

### PR DESCRIPTION
**Pull request by cnarans**

### CODE REVIEW

File: lib/reverse_characters.rb [hunk 1]
```diff
@@ -1,2 +1,8 @@
 def reverse_characters(str)
+    old = str.chars()
+    y=""
+    for i in 0..str.length-1
+        y=y+old.pop()
```
Comment (halfghaninne) [inline lib/reverse_characters.rb]: Great! In Ruby, we don't _have_ to include the `()`s when calling a method, unless we are passing in parameters.

Also, is there an operator that you can use as shorthand for this kind of operation: `some_var = some_var + something_else`?

File: lib/reverse_words.rb [hunk 2]
```diff
@@ -1,2 +1,8 @@
 def reverse_words(str)
+    x = str.split(" ")
+    y = ""
+    for i in 0..x.length-1
+        y = y + x.pop + " "
+    end
+    return y.slice(0, y.length-1)
```
Comment (halfghaninne) [inline lib/reverse_words.rb]: Hm. Nice fix, but I think it'd be more ideal if that trailing white space was not added to the value you want to return in the first place. Considering the data type of `y` might be helpful.

**Comment by cnarans** [comment]
Do this approach look better?

**Comment by halfghaninne** [comment]
👍

**Pull request closed without merging**